harvard-lts/jstor_aspace_oai_harvester
# dependabot updates

**Dependabot updates.**
* * *

**JIRA Ticket**: [(link)](https://at-harvard.atlassian.net/browse/LTSK8S-495)

# What does this Pull Request do?
Updating versions in requirements.txt, for dependabot alerts, covering high and moderate (no criticals).

# How should this be tested?
Note, already tested locally, deployed to dev and jenkins passed

- Checkout this version
- Start docker, it should start successfully
- Exec into docker and run pytest (done)
- Deploy to dev, jenkins/itest should pass (done)

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? N
- integration tests? N

# Interested parties
Tag (@ mention) interested parties
